test/test_config.yaml

```diff
@@ -0,0 +1,30 @@
+schemaVersion: "1.0.0"
+commandTests:
+- name: 'Default directory'
+  command: 'pwd'
+  expectedOutput: ['/app\n']
+
+- name: 'Node Installation Directory'
+  command: 'which'
+  flags: ['node']
+  expectedOutput: ['/nodejs/bin/node\n']
+
+- name: 'Node Version'
+  command: 'node'
+  flags: ['-v']
+  expectedOutput: ['v6.9.1\n']
+
+- name: 'Custom Node Version'
```

Comment:
this test uses the setup and teardown fields, which are part of https://github.com/GoogleCloudPlatform/runtimes-common/pull/49. these tests will fail under this PR is merged and deployed